codegen/src/api/mod.rs

```diff
@@ -253,7 +253,7 @@ impl RuntimeGenerator {
                 // Runtime APIs in the metadata by name.
                 pub static RUNTIME_APIS: [&str; #runtime_api_names_len] = [ #(#runtime_api_names,)* ];
 
-                /// The error type returned when there is a runtime issue.
+                /// The error type is returned when there is a runtime issue.
```

Comment:
```suggestion
                /// The error type that is returned when there is a runtime issue.
```

If you add "is"  here I think you need a little more :)

It's describing what the type is rather than what happens.